### PR TITLE
Flatpak updates

### PR DESCRIPTION
**Pull request by Hofer-Julian**
1. Removes flatpak folder
2. Updates `flatpak-install` rule

Further descriptions are in the commit messages.

If it's not too much work, a minor release would be appreciated after this PR has been merged

### CODE REVIEW

**Comment by m3nu** [comment]
No problem. We're in the middle of releasing anyways.

**Comment by m3nu** [comment]
`make bump-version` has a line to add the current version to Flatpak. This could need adjusting too?

**Comment by Hofer-Julian** [comment]
> `make bump-version` has a line to add the current version to Flatpak. This could need adjusting too?

You mean bumping the version in the xml?
That's fine, we are still going to use that. As a sidenote, it's called flatpak xml here, but I am pretty sure all distros use it that package vorta.

**Comment by real-yfprojects** [comment]
> The building happens at flathub, and we don't use this one for anything at the moment

... and the build instructions are stored in the flathub repository. So why keep the `flatpak-install` make command when it can only be used with assets that aren't stored in this repo? Also this make command is not referenced any longer: https://github.com/flathub/com.borgbase.Vorta/commit/10f3bb518000eee4de7641749fe16271d2fdfe8f

**Comment by Hofer-Julian** [comment]
> So why keep the flatpak-install make command when it can only be used with assets that aren't stored in this repo?

I have a slight preference to keep it here, but I don't mind either way

> Also this make command is not referenced any longer: 

I would reference it again after this PR is merged and released

**Review by real-yfprojects — CHANGES_REQUESTED** [review]

File: Makefile [hunk 1]
```diff
@@ -50,11 +50,11 @@ translations-to-qm:  ## Compile .ts text files to binary .qm files.
 	for f in $$(ls ${VORTA_SRC}/i18n/ts/vorta.*.ts); do lrelease $$f -qm ${VORTA_SRC}/i18n/qm/$$(basename $$f .ts).qm; done
 
 flatpak-install: translations-to-qm
-	pip3 install --prefix=/app --no-deps .
-	install -D ${FLATPAK_XML} /app/share/metainfo/com.borgbase.Vorta.appdata.xml
```
[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply Makefile:54]: Ah good catch, that was unintentional.
Will fix right away.

[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply Makefile:54]: I've also renamed the misleading `FLATPAK_XML` to `APPSTREAM_METADATA`.

**Review by real-yfprojects — APPROVED** [review]
Looks fine to me.

**Pull request merged**